Tools/WinMLDashboard/public/convert.py

```diff
@@ -31,6 +32,14 @@ def get_opset(ONNXVersion):
         return 8
     elif '1.5' == ONNXVersion:
         return 10
+    elif '1.6' == ONNXVersion:
+        return 11
+    elif '1.7' == ONNXVersion:
+        return 12
+    elif '1.8' == ONNXVersion:
+        return 13
+    elif '1.9' == ONNXVersion:
+        return 14
     else:
         return 7
```

Comment:
Maybe warn about the fallback:

```py
def get_opset(onnx_version):
    version_to_opset = {
        '1.2': 7,
        '1.3': 8,
        '1.5': 10,
        '1.6': 11,
        '1.7': 12,
        '1.8': 13,
        '1.9': 14,
    }
    opset = version_to_opset.get(onnx_version)
    if not opset:
        # print some warning
        return 7
    return opset
```

(Also, is there no opset 9?)

Reply:
I believe there should be? I don't have the context of why it's skipped here so I didn't try to add it, maybe someone in the team can fill me in here?

Reply:
Added v9 support